src/main/java/org/jitsi/videobridge/cc/SimulcastController.java

```diff
@@ -1205,41 +1234,81 @@ private boolean isSuspended()
             private int srcSeqNumStart = -1;
 
             /**
+             * Return whether or not the current frame is being accepted.
+             * srcSeqNumLimit would not be updated (remain as -1) if the
+             * frame were either:
+             * a) an old frame of an adaptive stream OR
+             * b) a frame from an adaptive stream on a layer we're not using
+             * @return true if we're accepting this frame
+             */
+            private boolean isAccepting()
+            {
+                return srcSeqNumLimit != -1;
+            }
+
+            /**
+             * Update this {@link SeenFrame} with information from its
+             * corresponding {@link FrameDesc}
+             * @param frame the {@link FrameDesc} from which to update
+             */
+            void update(FrameDesc frame)
```

Comment:
I think this could be renamed to something more descriptive than "update." (Ignore if people that would be reading this generally would know what that means.)